packages/integration-tests/api/executions.js

```diff
@@ -10,16 +10,15 @@ const { callCumulusApi } = require('./api');
  * @param {string} params.arn - an execution arn
  * @returns {Promise<Object>} - the execution fetched by the API
  */
-async function getExecution({ prefix, arn }) {
-  return callCumulusApi({
+const getExecution = ({ prefix, arn }) =>
```

Comment:
I realize that the function doesn't need to be marked as `async`, but is there some reason to refer the `const` style definition for functions?

Reply:
In this case it's personal preference because it allows for the more compact "arrow function without an explicit return" syntax.